tasks/1_golf-score/golf-score.js

```diff
@@ -0,0 +1,19 @@
+export const GOLF_SCORES = {
+  HOLE_IN_ONE: `Hole_in_one`,
+  EAGLE: `Eagle`,
+  BIRDIE: `Birdie`,
+  PAR: `Par`,
+  BOGEY: `Bogey`,
+  DOUBLE_BOGEY: `Double Bogey`,
+  GO_HOME: `Go Home`,
+};
+
+export function golfScore(par, strokes) {
+  if (strokes == 1) return GOLF_SCORES.HOLE_IN_ONE;
+  else if (strokes <= par - 2) return GOLF_SCORES.EAGLE;
+  else if (strokes == par - 1) return GOLF_SCORES.BIRDIE;
+  else if (strokes == par) return GOLF_SCORES.PAR;
+  else if (strokes == par + 1) return GOLF_SCORES.BOGEY;
+  else if (strokes == par + 2) return GOLF_SCORES.DOUBLE_BOGEY;
```

Comment:
you should use triple equals (===)
https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a